datasetinsights/io/config_handler.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import os
+import tempfile
+
+from yacs.config import CfgNode
+
+from datasetinsights.constants import GCS_BASE_STR
+from datasetinsights.io import create_downloader
+
+logger = logging.getLogger(__name__)
+
+
+class ConfigHandler:
+    """ Handles all the config related tasks such as
+     loading config from local or remote locations.
+    """
+
+    @staticmethod
+    def load_config(config):
```

Comment:
I think it's better to rename variable here to `def load_config(path)`. Also the docstring should mention what type of paths are supported. Bonus point if some examples are provided. 

Did you update the corresponding files in `docs/` so that the docstring can be updated properly?

Reply:
I will update.  I have missed to update for Mlflow change also, I will include in this commit.